src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/TypeScript/azure-devops-report/vss-extension.json

```diff
@@ -25,8 +25,26 @@
     "content": {
         "details": {
             "path": "overview.md"
+        },
+        "license": {
+            "path": "LICENSE"
+        }
+    },
+    "links": {
+        "getstarted": {
+            "uri": "https://learn.microsoft.com/en-us/dotnet/ai/quickstarts/evaluate-ai-response"
+        },
+        "learn": {
+            "uri": "https://learn.microsoft.com/en-us/dotnet/ai/conceptual/evaluation-libraries"
```

Comment:
It may be nice to include the samples link in this pane here as well

Reply:
I thought about that but these are not links we can specify. There are actually a set of predetermined keys that are accepted here. There is not one for samples.